lib/ssl/doc/src/ssl.xml

```diff
@@ -1231,6 +1231,33 @@ fun(srp, Username :: binary(), UserState :: term()) ->
 	</desc>
       </datatype>
 
+      <datatype>
+        <name name="use_srtp_protection_profiles"/>
+        <desc>
+          <p>Configures the list of the client's acceptable SRTP Protection Profiles.</p>
+          <p>In order to negotiate the use of SRTP data protection, clients
+          include an extension of type "use_srtp" in the DTLS extended client
+          hello.  This extension MUST only be used when the data being
+          transported is RTP or RTCP.</p>
+          <p>This extension MUST only be used with DTLS, and not with TLS</p>
+        </desc>
+      </datatype>
+
+      <datatype>
+        <name name="use_srtp_mki"/>
+        <desc>
+          <p>Configures the SRTP Master Key Identifier chosen by the client.</p>
+          <p>This options only has effect only when <c>use_srtp_protection_profiles</c>
+          option is specified.</p>
+          <p>The srtp_mki field contains the value of the SRTP MKI which is associated
+          with the SRTP master keys derived from this handshake.  Each SRTP
+          session MUST have exactly one master key that is used to protect
+          packets at any given time.  The client MUST choose the MKI value so
+          that it is distinct from the last MKI value that was used, and it
+          SHOULD make these values unique for the duration of the TLS session.</p>
+        </desc>
+      </datatype>
+
```

Comment:
It should be mentioned that the user is required to handle SRTP via the via a custom backend so that
nobody expects this to work of shelf.